homeassistant/components/yolink/sensor.py

```diff
@@ -10,31 +13,46 @@
     SensorStateClass,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_STATE, PERCENTAGE
-from homeassistant.core import HomeAssistant
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.util import percentage
 
+from . import YoLinkCoordinator
 from .const import (
     ATTR_CLIENT,
+    ATTR_COORDINATOR,
     ATTR_DEVICE,
     ATTR_DEVICE_DOOR_SENSOR,
+    ATTR_DEVICE_ID,
+    ATTR_DEVICE_NAME,
     ATTR_DEVICE_TYPE,
-    ATTR_MQTT_CLIENT,
-    ATTR_PLATFORM_SENSOR,
     DOMAIN,
 )
 from .entity import YoLinkEntity
 
-SENSOR_DESC_MAP: dict[str, SensorEntityDescription] = {
-    "battery": SensorEntityDescription(
+
+@dataclass
+class YoLinkSensorEntityDescription(SensorEntityDescription):
+    """YoLink SensorEntityDescription."""
+
+    value: Callable = round
```

Comment:
Are we sure that we want to round by default? An alternative is to just return the state as is.
```suggestion
    value: Callable = lambda state: state
```